docs/source/architecture/servicefees.rst

```diff
@@ -0,0 +1,83 @@
+.. _service-fees:
+
+Service Fees (Pricing)
+======================
+
+Minimum fee rate
+----------------
+
+When they join the network, stakers specify a minimum fee rate, on a *per sharing policy* and *per 24h period* basis, that their worker (Ursula) will accept at the point of engagement with a network user (Alice). If Alice’s offered per period rate (contained alongside the sharing policy’s parameters in an ``Arrangement`` object) for a specified policy duration computes as equal to or greater than the minimum fee rate, the sharing policy will be accepted by Ursula and the access control service will commence. Ursula will service the sharing policy by being online and answering access requests, at that unchanging per period fee rate, until the specified expiration date or an early revocation is instigated by Alice.
+
+The minimum fee rate applies to each individual worker machine managing a given sharing policy. In other words, the rate is also *per Ursula*. 
+
+If Alice wishes to employ multiple Ursulas to service a single sharing policy (``n`` > 1), a common configuration, then they must pay each staker associated with that policy the same exact fee rate. This rate will be the highest from the set of *minimum* fee rates specified by the stakers with which they engage. Alices may attempt price optimization strategies to find the cheapest group of Ursulas.
+
+When issuing a sharing policy, Alices are required to escrow a deposit covering the cost of the entire duration of the policy. The deposit is paid out in tranches to stakers once per period, in the period after their Ursula makes a commitment (formerly referred to as *confirm activity*). Therefore, the minimum sum of fees a staker can receive each period is the product of their specified minimum fee rate and the number of active sharing policies their Ursula has been assigned.
+
+
+Global fee range
+----------------
+
+The global fee range is a means of establishing quasi-universal pricing for the NuCypher service. It is enforced via the function ``feeRateRange`` in ``PolicyManager.sol``, which specifies per sharing policy and per 24h period constraints expressed in **WEI**. In other material, rates are discussed in **GWEI** (and fiat).
+
+The minimum fee rate chosen by stakers must fall within the global fee range. The network will launch with the following parameters:
+
+**Minimum fee rate**
+
+XX GWEI (~$X.XX x10^-X) *per period*
+
+XX,XXX GWEI ($X.XXX) *per year*
+
+**Maximum fee rate**
```

Comment:
The term "maximum fee rate" seems to imply it's an absolute upper bound to the fee rate, while what we have is an upper bound on the _offered_ rate. Or in other words, nothing prevents Alice from paying a price higher than this rate; this may be particularly relevant with handpicked Ursulas that offer a premium service.

Reply:
@vzotova warned me about this a few months ago and it slipped my mind 😏 
We can add a caveat and change the naming slightly. I do believe this is a quite extreme edge case tho 

- As it stands, any rate specified by a staker that's >max will be changed and shown to Alices as the default rate (in my draft pricing, this figure is quite a lot lower). 
- They then have to accept that rate for their service for any customer that comes along, regardless of any premium feature or differentiating attribute, and regardless of the existence of other Alices that are willing to pay more 
- So within the current rules, if you think there are Alices willing to pay 'more than normal' for your specific Ursula, it makes more sense to choose the max permitted rate and/or lobby for a widening of the range.

On Alice's side, while it's possible to posit a >max rate in their Arrangement, there are some reasons why they are unlikely to do so within 'normal' behaviour.
- One cannot easily identify Ursulas with premium services using their specified rate alone, unless they equal or are close to the max
- If stakers are restrained by the fee range, paying them more than needed is probably part of a rare strategy or trustful/dodgy relationship (e.g. cycling money around to keep that particular staker running) 
- In general, requiring multiple Ursulas for a single policy could make benefitting from unique services a high friction exercise

Reply:
```suggestion
**Maximum chargeable fee rate**
```